internal/querynode/query_node.go

```diff
@@ -328,7 +328,6 @@ func (node *QueryNode) Stop() error {
 		if err != nil {
 			log.Warn("session fail to go stopping state", zap.Error(err))
 		} else {
-			node.UpdateStateCode(commonpb.StateCode_Stopping)
```

Comment:
Why remove the stopping state here

Reply:
it repeated.
![image](https://user-images.githubusercontent.com/21985684/209645423-8fbec22d-6b8e-4460-aed1-b2602d363c82.png)